src/api/helpers/map-CIS-countries/map-CIS-country/no-insurance-support/index.ts

```diff
@@ -0,0 +1,43 @@
+import { EXTERNAL_API_DEFINITIONS } from '../../../../constants';
+import countryRatingIsAorB from '../country-rating-is-a-or-b';
+import countryRatingIsCorD from '../country-rating-is-c-or-d';
+import esraClassificationIsStandardHighOrVeryHigh from '../esra-classification-is-standard-high-or-very-high';
+import { NoInsuranceSupportParams } from '../../../../types';
+
+const {
+  CIS: {
+    COUNTRY_RATINGS: { NOT_APPLICABLE },
+    ESRA_CLASSIFICATION: { NONE },
+    SHORT_TERM_COVER: { UNLISTED, CILC },
+  },
+} = EXTERNAL_API_DEFINITIONS;
+
+/**
+ * noInsuranceSupport
+ * Check if a country has no insurance support (online or offline)
+ * @param {String} countryRating: Country rating
+ * @param {String} esraClassification: ESRA classification
+ * @param {String} shortTermCover: Short term cover
+ * @returns {Boolean}
+ */
+const noInsuranceSupport = ({ countryRating, esraClassification, shortTermCover }: NoInsuranceSupportParams): boolean => {
+  const shortTermCoverIsUnlisted = shortTermCover === UNLISTED;
+  const esraClassificationIsNone = esraClassification === NONE;
+  const countryRatingIsNotApplicable = countryRating === NOT_APPLICABLE;
+
+  if (
+    shortTermCoverIsUnlisted &&
+    esraClassificationIsNone &&
+    (countryRatingIsAorB(countryRating) || countryRatingIsCorD(countryRating) || countryRatingIsNotApplicable)
```

Comment:
could this be another constant? easier to read

Reply:
I think either works - personally think new consts could be confusing to others. Have moved 2x conditions into their own const's